docs/Deploying-Pantheon/High-Availability.md

```diff
@@ -0,0 +1,102 @@
+description: Pantheon high availability 
+<!--- END of page meta data -->
+
+# High Availability 
+
+To enable high availability to the [RPC Pub/Sub API over WebSockets](../Pantheon-API/RPC-PubSub.md) 
+or the [JSON-RPC API](../Pantheon-API/Using-JSON-RPC-API.md) run and synchronize multiple Pantheon 
+nodes to the network. Use a load balancer to distribute requests across nodes in the cluster that 
+are ready to receive requests. 
+
+## Determining When a Node is Ready 
+
+To determine when a node is ready use: 
+
+* [`net_peerCount`](../Reference/Pantheon-API-Methods.md#net_peercount) to confirm the minimum number of peers has been reached
+* [`eth_syncing`](../Reference/Pantheon-API-Methods.md#eth_syncing) to confirm the node is within the required number of blocks 
+from the best known block for that node 
+
+!!! note
+    The minimum number of peers and number of blocks from the best known block for a node to be considered ready
+    is deployment specific. 
+
+## Transaction Nonces 
+
+The account nonce for the next transaction is obtained using [`eth_getTransactionCount`](../Reference/Pantheon-API-Methods.md#eth_gettransactioncount). 
+The account nonce depends on the transactions in the [transaction pool](../Using-Pantheon/Transactions/Transaction-Pool.md).
+If [`eth_getTransactionCount`](../Reference/Pantheon-API-Methods.md#eth_gettransactioncount) and 
+[`eth_sendRawTransaction`](../Reference/Pantheon-API-Methods.md#eth_sendrawtransaction) requests for a specific account 
+are sent to multiple nodes, the [`eth_getTransactionCount`](../Reference/Pantheon-API-Methods.md#eth_gettransactioncount)
+results can be incorrect. 
+
+!!! note
+    If using [private transactions](../Privacy/Privacy-Overview.md), `eea_getTransactionCount` is used to obtain 
+    the account nonce and [`eea_sendRawTransaction`](../Reference/Pantheon-API-Methods.md#eea_sendrawtransaction)
+    to send private transactions. 
+
+To get correct account nonces when distributing requests across a cluster, do one of:  
+
+* Track the next nonce outside of the Pantheon node (as MetaMask does)
+* Configure the load balancer in sticky mode so requests from a specific account are sent to a single 
+node unless that node is unavailable. 
```

Comment:
@joshuafernandes Is "sticky mode" a real term or just a colloquialism I picked up from my last employer?

Reply:
sticky sessions mode looks legit to me. It's even in AWS docs.